### PR TITLE
Dashboards postgres & rabbitmq & elasticsearch

### PR DESCRIPTION
**Pull request by SlimaneAmar**

### CODE REVIEW

File: docker-compose/technical/docker-compose.technical.yml [hunk 1]
```diff
@@ -45,6 +45,17 @@ services:
...
 
+  postgres-exporter:
+    profiles:
+      - all
+      - metrics
+    image: prometheuscommunity/postgres-exporter:v0.15.0
+    ports:
+      - "9187:9187"
+    environment:
+      - DATA_SOURCE_NAME=postgresql://postgres:postgres@postgres:5432/postgres?sslmode=disable
```
Comment (Tristan-WorkGH) [inline docker-compose/technical/docker-compose.technical.yml:56]: in [the documentation](https://github.com/prometheus-community/postgres_exporter#environment-variables), it say
> DATA_SOURCE_NAME the default legacy format. Accepts URI form and key=value form arguments. The URI may contain the username and password to connect with.

Wouldn't it be better not to use legacy parameters even if it's the default?
```suggestion
      DATA_SOURCE_USER: postgres
      DATA_SOURCE_PASS: postgres
      DATA_SOURCE_URI: postgres:5432/postgres?sslmode=disable
```

**Review by Tristan-WorkGH — CHANGES_REQUESTED** [review]
1 missing config + 1 question + 1 suggestion

File: docker-compose/technical/docker-compose.technical.yml [hunk 2]
```diff
@@ -45,6 +45,17 @@ services:
       - $PWD/../technical/init-merging-data.sh:/init-merging-data.sh:Z
     restart: unless-stopped
 
+  postgres-exporter:
+    profiles:
+      - all
+      - metrics
+    image: prometheuscommunity/postgres-exporter:v0.15.0
+    ports:
+      - "9187:9187"
```
[the remark this answers is not in the document]
Comment (SlimaneAmar) [reply docker-compose/technical/docker-compose.technical.yml:54]: Yes for tests
Only for dev

File: docker-compose/technical/docker-compose.technical.yml [hunk 3]
```diff
@@ -45,6 +45,17 @@ services:
...
+    environment:
+      - DATA_SOURCE_NAME=postgresql://postgres:postgres@postgres:5432/postgres?sslmode=disable
+    restart: unless-stopped
+
```
[the remark this answers is not in the document]
Comment (SlimaneAmar) [reply docker-compose/technical/docker-compose.technical.yml:58]: Done

**Review by Tristan-WorkGH — APPROVED** [review]

**Review by antoinebhs — APPROVED** [review]
Code OK
Test OK

I mention here that the JSON dashboards are from:
* https://grafana.com/grafana/dashboards/14191-elasticsearch-overview/,
* https://grafana.com/grafana/dashboards/10991-rabbitmq-overview/, 
* https://grafana.com/grafana/dashboards/9628-postgresql-database/

They will be later extended with alerts.

**Pull request merged**